bundles/org.openhab.binding.ferroamp/README.md

```diff
@@ -0,0 +1,222 @@
+| eso-ubat             | Number:ElectricPotential   | R          | Measured on battery side                 |
+| eso-relaystatus      | String                     | R          | 0 = relay closed (i.e running power), 1 = relay open/disconnected, 2 = precharge|
+| eso-soc              | Number:Dimensionless       | R          | State of Charge for ESO                  |
+| eso-temp             | Number:Temperature         | R          | Measured inside ESO                      |
+| eso-wpbatcons        | Number:Energy              | R          | Total energy produced by ESO, i.e total energy charged|
+| eso-wbatprod         | Number:Energy              | R          | Total energy produced by ESO, i.e total energy discharged|
+| eso-udc              | Number:Power               | R          | DC link voltage as measured by ESO       |
+| eso-ts               | DateTime                   | R          | Time stamp when message was published    |
+| esm-soh              | Number:Dimensionless       | R          |                                          |
+| esm-soc              | Number:Dimensionless       | R          |                                          |
+| esm-rated-capacity   | Number:Energy              | R          | Rated capacity of battery                |
+| esm-id               | String                     | R          | Unique identifier of battery. If available, this will be the unique id that the battery reports. |
+| esm-rated-power      | Number:Power               | R          | Rated power of battery                   |
+| esm-status           | String                     | R          | Dependent on battery manufacturer        |
+| esm-ts               | DateTime                   | R          | Time stamp when message was published    |
+
+The following channels are available for `Ferroamp` EnergyHub configuration. Please, see Ferroamp documentation for more details.
+
+| Channel Type ID      | Item Type                  | Read/Write | Description                              |
+|---------------------------------------------------|------------|------------------------------------------|
+| request-charge       | String                     | W          | Set charge power, value in Watt          |
+| request-discharge    | String                     | W          | Set discharge power, value in Watt       |
+| request-auto         | String                     | W          | Set auto power. Returning control of batteries to system, value as auto.|
+
+# Full Example
+
+## Thing
+
+```java
+Thing ferroamp:energyhub:myenergyhub [ hostName="energyhub-ip", userName="myUserName", password="myPassword", hasBattery=false ]
+```
+
+## Items
+
+```java
+Number:Energy ferroamp "ehub-wloadconsq-l1" <energy> { channel="ferroamp:energyhub:myenergyhub:ehub-wloadconsq-l1" }
+String ferroamp "request-charge" <energy> { channel="ferroamp:energyhub:myenergyhub:request-charge" }
```

Comment:
These items have the same name, that is not going to work.

Reply:
Done changes 20240607

Reply:
Not fixed.